web/src/main/java/pl/edu/agh/papaya/rest/login/controller/LoginController.java

```diff
@@ -0,0 +1,26 @@
+package pl.edu.agh.papaya.rest.login.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import pl.edu.agh.papaya.api.model.LoginRequest;
+import pl.edu.agh.papaya.api.model.LoginResult;
+import pl.edu.agh.papaya.api.service.LoginApi;
+import pl.edu.agh.papaya.security.AuthenticationService;
+
+@RestController
+public class LoginController implements LoginApi {
+
+    @Autowired
+    private AuthenticationService authenticationService;
+
+    @Override
+    public ResponseEntity<LoginResult> requestLogin(LoginRequest request) {
```

Comment:
W api można by edytować, żeby nie było tego pola `admin`, a `username` mogłoby mieć więcej sensu jako email - bo tym tak naprawdę jest (chociaż w świecie autentykacji jest oczywiście nazwą użytkownika)

Reply:
Raczej tutaj nie mogę zmienić nazwy `username` bo tak to jest traktowane przez spring security. Ewentualnie możemy na razie pozwalać tylko na emaile jako username i tyle.

Reply:
No wiem, że generalnie funkcjonowałoby jako username - myślałam ewentualnie o samym polu w `LoginRequest`

Reply:
No właśnie nie jestem pewny też co do tego. Jestem za tym, żeby na razie to zostawić jako username wszędzie, a przy okazji taska z logowaniem w kolejnym sprincie jak już wybierzemy jakiś system logowania to się wyklaruje jak to powinno wyglądać.

Reply:
Znaczy moja sugestia brała się z tego, że informacja z pola `LoginRequest.username` i tak będzie musiała trafić do `User.email` generalnie, ale nie jest to jakieś super istotne

Reply:
Jak dla mnie takie coś jak najbardziej wpisuje się w styl biedaloginu :slightly_smiling_face: